admin/src/components/Events.tsx

```diff
@@ -46,6 +46,23 @@ function EventCard(props: {
       ? "Normal"
       : "Hard";
 
+  const categoryType = (() => {
+    switch (props.event.category as string) {
+      case "FOOD":
+        return "Food";
+      case "NATURE":
+        return "Nature";
+      case "HISTORICAL":
+        return "Historical";
+      case "CAFE":
+        return "Cafe";
+      case "DININGHALL":
+        return "Dininghall";
+      default:
+        return "Dorm";
```

Comment:
You can just subscript the string after the first letter to set lowercase rather than this entire switch statement